docs/versioning.md

```diff
@@ -0,0 +1,52 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+
+# Versioning
+
+`tezos-packaging` follows all Octez releases, both stable and candidates.
+
+Packages from our releases provide addinional functionality (e.g. systemd services
+in Ubuntu and Fedora package or brew formulae with launchd services).
+This additional functionality may change within the same upstream version
+
+In order to track these additional functionality chages, our GitHub releases and packages use
```

Comment:
Small suggestion:
```suggestion
In order to track this, our GitHub releases and packages use
```
for two reasons:
1. "changes" was misspelled
2. "additional functionality" is in both the previous two sentences as well as in one of the points just following, I think it's worth removing _some_ of this repetition :sweat_smile: